### DISCUSSION TITLE
[FEATURE] : Make a contributors section

### DISCUSSION THREAD

**Discussion by PritamSarbajna** [PritamSarbajna/tourism-website#17] [Ideas]
We can make a contributors section where we can list the contributors of this project. I was thinking a carousel with their photo, name and github username. How about this ?

**Comment by 3xp11i** [comment]
It will be great! We can work on that together

**Reply by PritamSarbajna** [reply]
Hey @prakhartiwari2507 co-author in this issue #18

**Reply by 3xp11i** [reply]
How can I do so? Can you please guide me? I am not much experienced with the functionality of GitHub.

**Reply by PritamSarbajna** [reply]
yeah sure I'm providing you some tutorial

**Reply by 3xp11i** [reply]
Sure, I will be waiting.

**Comment by PritamSarbajna** [comment]
Hey @prakhartiwari2507 just see this you need to make more or less like this [website](https://youtu.be/daAVTmsMXeI)
Remember you don't need to add the hover effects I just need the layout when it is not on hover.
Include these things only :
- profile photo in a circle shape
- Name
- Their github profile link in a github icon

You can find contributors [here](https://github.com/PritamSarbajna/tourism-website/graphs/contributors)

Also let me know if you need any help and if you're finding it too difficult

**Reply by 3xp11i** [reply]
Sure! I will try my best, once I have made this box I will send you the HTML and CSS code.

**Reply by 3xp11i** [reply]
Hey @PritamSarbajna, 
Is this what you wanted?
Let me know if you want any changes!
![image](https://user-images.githubusercontent.com/65062036/184992321-b74e06da-512a-48f5-8aaf-034187722432.png)


[Contribution Card (No Effects + Unresponsive).zip](https://github.com/PritamSarbajna/tourism-website/files/9354915/Contribution.Card.No.Effects.%2B.Unresponsive.zip)

**Reply by PritamSarbajna** [reply]
Hey @prakhartiwari2507 just remove `web developer | coder` and also make the github text a github logo so that when we press on the github icon we visit the contributor's github profile

**Reply by 3xp11i** [reply]
Buddy It is easy, we just need to remove the h3 tag and its styling, and yeah I have made the GitHub Icon as a link which redirects to the contributor's GitHub Profile.

**Reply by 3xp11i** [reply]
Have you downloaded and checked the website yourself? @PritamSarbajna

**Reply by 3xp11i** [reply]
**Check these files** @PritamSarbajna 

[Contribution Card.zip](https://github.com/PritamSarbajna/tourism-website/files/9365138/Contribution.Card.zip)

https://user-images.githubusercontent.com/65062036/185211831-b65b0950-ea4e-4ae5-ad01-fcd61cf17a44.mp4

**Reply by PritamSarbajna** [reply]
Hey @prakhartiwari2507 this is not a good design please try to make it little small add hover effects and all

**Reply by 3xp11i** [reply]
Buddy, you only said to not add hover effects. And can you tell me how small you want it to be? I can't just work on estimates, it will waste the time of both of us. Kindly tell me the dimensions and the kind of hover effect you want.

**Reply by PritamSarbajna** [reply]
Hey @prakhartiwari2507 I have said not to add the hover effect in the tutorial you can change the color when in hover. So let me make this easy for you. Just consider the following design :

![Add a subheading](https://user-images.githubusercontent.com/90236635/185435933-e2bbfde3-421c-427d-a737-bb4c2987519f.png)

> Ignore the white background

**Reply by PritamSarbajna** [reply]
And also keep in mind that picture of the contributors should be the picture they have on github profile

**Reply by 3xp11i** [reply]
Okay, buddy, I have created a hover effect on the logo and name, kindly tell me the dimensions of the profile picture. I will learn about APIs to integrate a user's profile photo. I will share it when completed.

**Reply by 3xp11i** [reply]
First test the code on your side. And please watch out that I haven't used any file for the profile photo, it is a GitHub Link in which the username will be replaced by the username of the contributor. 
+ I have also made a hover effect on the logo which is a link to open the contributor's profile.

[Card.zip](https://github.com/PritamSarbajna/tourism-website/files/9379564/Card.zip)

https://user-images.githubusercontent.com/65062036/185553947-fb369020-96e8-4339-90bd-ae3ca8f11793.mp4

**Reply by PritamSarbajna** [reply]
make a PR

**Reply by 3xp11i** [reply]
How? I mean should I add a new page or section to the website? Or should I just keep this contribution card thing in a separate folder? What to do and how to do please explain.

**Reply by PritamSarbajna** [reply]
Yeah @prakhartiwari2507 just make a section about it before the contact us section and make it a carousel.

**Reply by 3xp11i** [reply]
How many people do I have to add?

**Reply by PritamSarbajna** [reply]
Hey @prakhartiwari2507 don't use this kind of word to address someone it's so unprofessional also you can see the contributors [here](https://github.com/PritamSarbajna/tourism-website/graphs/contributors)

Thank you

**Reply by 3xp11i** [reply]
I am sorry again, but may I please clear my confusion and be clear to know what to do? :)

**Reply by PritamSarbajna** [reply]
Yeah sure. You have to make this card for every contributors and then use it as a carousel.
> If you don't know what a carousel is then [here is a bootstrap carousel](https://getbootstrap.com/docs/5.2/components/carousel/)
>
> Note : *Please don't use bootstrap links or something it's just for reference you can watch the tutorials to make a carousel with pure css*
